apidef/migration.go

```diff
@@ -13,30 +15,45 @@ func (a *APIDefinition) MigrateVersioning() (versions []APIDefinition, err error
 	}
 
 	if a.VersionData.NotVersioned && len(a.VersionData.Versions) > 1 {
-		return nil, errors.New("not migratable - if not versioned, there should be 1 version info in versions map")
+		return nil, errors.New("not migratable - if not versioned, there should be just one version info in versions map")
```

Comment:
can you change `not migratable` to `This API can not be migrated because it doesn't have any version. At least one version is required so that the migration can happen.`